solution-base/build.sh

```diff
@@ -37,6 +37,8 @@ MONGODB_SINGLE_NODE_PATH=${ISO_ROOT}/deploy/mongodb-1-node.yaml
 MONGODB_THREE_NODE_PATH=${ISO_ROOT}/deploy/mongodb-3-nodes.yaml
 MONGODB_SHARDED_SINGLE_NODE_PATH=${ISO_ROOT}/deploy/mongodb-sharded-1-node.yaml
 MONGODB_SHARDED_THREE_NODE_PATH=${ISO_ROOT}/deploy/mongodb-sharded-3-nodes.yaml
+MONGODB_SHARDED_SINGLE_NODE_THREE_SHARDS_PATH=${ISO_ROOT}/deploy/mongodb-sharded-1-node-3-shards.yaml
+MONGODB_SHARDED_THREE_NODE_THREE_SHARDS_PATH=${ISO_ROOT}/deploy/mongodb-sharded-3-nodes-3-shards.yaml
```

Comment:
I am interested in having a bit more context, this will create two new yaml files for MongoDB, each containing multiple resources, right? But I am not able to see where it is used in https://github.com/scality/artesca/pull/1721/files#diff-9f3a8c4af71c5060627586aa78119e38ab963af820e4b554b5d204c369f8534cR28 (more specifically, how the nShards is linked at install and upgrade time).

I am asking because my PR to handle mongodb upgrade must also handle this case, but I do see logic in the Zenko.py file still relying on only one shard (like in https://github.com/scality/artesca/blob/e16707350054fbfa6980131206358ff57d4075c6/installer/artesca_installer/salt/_modules/zenko.py#L158), am I missing something?